R/sce_to_anndata.R

```diff
@@ -20,7 +23,7 @@
 #'   anndata_file = "test_anndata.h5"
 #' )
 #' }
-sce_to_anndata <- function(sce, anndata_file, x_assay_name = "counts") {
+sce_to_anndata <- function(sce, anndata_file, x_assay_name = "counts", compression = "gzip", ...) {
```

Comment:
Updating spacing, and preparing for more formal argument checking.
```suggestion
sce_to_anndata <- function(
  sce, 
  anndata_file, 
  x_assay_name = "counts", 
  compression = c("gzip", "none", "lzf"), 
  ...
) {
```